build/webpack.base.config.js

```diff
@@ -107,7 +113,7 @@ module.exports = {
                     {
                         test: /\.scss$/,
                         use: [
-                            ...getCssLoaders(config),
+                            ...getCssLoaders(config, 1),
```

Comment:
`sass-loader` comes after the css loaders so we pass 1 as the additional loaders.